docs/reference/search/search.asciidoc

```diff
@@ -550,6 +550,18 @@ Period of time used to extend the life of the PIT.
 (Optional, <<query-dsl,query object>>) Defines the search definition using the
 <<query-dsl,Query DSL>>.
 
+[[request-body-rank]]
+`rank`::
+(Optional, object) Defines a method for combining and ranking multiple queries
```

Comment:
I wonder if mentioning queries may be confusing, because we currently only allow to combine one or more knn with one query. I am merely referring to the fact that users may not see knn as a query because it's a different section of the search API.

Reply:
Agreed. I will change when I work on the docs in the last step.

Reply:
Updated.